i18n/src/i18n-messages.json

```diff
@@ -445,11 +445,11 @@
   },
   {
     "id": "verify-identity.vetting_freja_tagline",
-    "defaultMessage": "For those able to create a Freja eID by passing by one of the authorised agents"
+    "defaultMessage": "For those able to create a Freja eID by passing one of the authorised agents"
```

Comment:
I think that I misread this string in the previous review, I asked for the removal of "by" but I'd now say that I was wrong :-/

Reply:
updated!